app/javascript/controllers/conditional_question_controller.js

```diff
@@ -12,7 +12,6 @@ export default class extends Controller {
       case "radio":
         this.displayConditionalRadio()
       default:
-        console.log("Not yet implemented for " + this.element.type)
```

Comment:
Do we not want anything to happen on the default case?

Reply:
I don't think so, it's more there as a guard but we shouldn't have any conditionals that don't fall into one of the explicit cases (since we wouldn't know how to action them anyway)